Formula/tee-clc.rb

```diff
@@ -20,7 +20,6 @@ def install
   end
 
   test do
-    ENV["TF_ADDITIONAL_JAVA_ARGS"] = "-Duser.home=#{ENV["HOME"]}"
```

Comment:
are we sure this is passed through transparently by just setting it in _JAVA_OPTIONS

Reply:
Same comment as above. Tested it & the magic works.